packages/appstore/src/components/eu-disclaimer/__tests__/eu-disclaimer.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import EUDisclaimer from '../eu-disclaimer';
+
+const mockedRootStore = mockStore({});
+
+describe('<EUDisclaimer />', () => {
+    it('Check disclaimer exists', () => {
+        render(
+            <StoreProvider store={mockedRootStore}>
+                <EUDisclaimer />
+            </StoreProvider>
+        );
+
+        const eu_statutory_disclaimer = screen.queryByText('EU statutory disclaimer');
+
+        expect(eu_statutory_disclaimer).toBeInTheDocument();
```

Comment:
could we not use getByText here? why use queryByText

Reply:
@ameerul-deriv in line 17 we can use `getByText` since the text is there in the DOM, but when it's not there and we want to check it's absent, we need to use `queryByText`(line 29)